sdk/core/azure-core/inc/azure/core.hpp

```diff
@@ -39,6 +39,8 @@
 
 // azure/core/http
 #include "azure/core/http/http.hpp"
+#include "azure/core/http/http_status_code.hpp"
+#include "azure/core/http/raw_response.hpp"
```

Comment:
both are included in http.hpp.  We don't need them to be here.

Reply:
There are other header files in here that aren't absolutely necessary because one includes the other. But that relationship could change, so for simplicity (easier to maintain) we add all the public headers here that are not compile options dependent, like curl transport (except version.hpp).

For example, etag includes nullable, but we still include it explicitly.